.devcontainer/Dockerfile

```diff
@@ -0,0 +1,51 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Still a bit on the fence about Microsoft-copyrighted stuff in here, but it makes development easier.

Reply:
Yeah, I get that. In this case its just the MIT license on the dockerfile tho.

No different as our current license